src/main/webapp/script.js

```diff
@@ -19,75 +19,69 @@ google.charts.load('current', {
 });
 
 function passQuery() {
-  const information = document.getElementById('more-info');
-  information.innerText = 'Please wait. Loading...';
+  document.getElementById('result').style.display = 'block';
   const query = new FormData(document.getElementById('query-form'));
   const personType = query.get('person-type');
-  const action = query.get('action');
   const location = query.get('location');
 
-  const fetchUrl = '/query?person-type=' + personType +
-      '&action=' + action +
-      '&location=' + location;
-  
-  fetch(fetchUrl)
-    .then((response) => {
-      if (response.ok) {
-        response.json().then((jsonResponse) => JSON.parse(jsonResponse))
-        .then((data) => {
-          // data is a 2D array, where the first row is a header row and all
-          // subsequent rows are one piece of data (e.g. for a state or county)
-          information.innerText = '';
-          displayVisualization(data);
-        });
-      } else {
-        console.log(
-          `An error occurred: ${response.status}: ${response.statusText}`);
-      }
-    });
+  fetch('/query?person-type=' + personType + '&location=' + location)
+      .then((response) => {
+        if (response.ok) {
+          response.json().then((jsonResponse) => JSON.parse(jsonResponse))
+              .then((data) => {
+                // data is a 2D array, where the first row is a header row and all
+                // subsequent rows are one piece of data (e.g. for a state or county)
+                displayVisualization(data);
+              });
+        } else {
+          console.log('There was an error');
+        }
+      });
 }
 
-// displayVisualization takes in a data array representing the 2D array
-// returned  by the census API. The first row in the census Data array
-// is the header, which describes the type of data.
-// Eg: Header for the query that finds for populations of all counties is
-// ["NAME","S0201_001E","state","county"]
+// displayVisualization takes in a data array representing the 2D array returned by the
+// census API. The function creates the necessary visualization with the census data.
 function displayVisualization(censusDataArray) {
   if (censusDataArray[0][2] == 'state' && censusDataArray[0][3] != 'county') {
     google.charts.setOnLoadCallback(drawRegionsMap(censusDataArray));
   } else {
     // We currently do not have counties implemented
-    const errorMessage = 'We do not support this visualization yet';
     document.getElementById('map').innerHTML = '';
-    document.getElementById('more-info').innerHTML = errorMessage;
+    document.getElementById('more-info').innerHTML = 'We do not have this information yet.';
```

Comment:
It looks like some of the deletions/insertions are up to date but some aren't. Here it should be 'We do not support this visualization yet' but later in the code it shows more up to date insertions.

Reply:
I must have mistakenly started from an old version of master I had locally, fixed to match master